src/icrar/leap-accelerate/math/linear_math_helper.cc

```diff
@@ -48,6 +48,19 @@ namespace icrar
         return res;
     }
 
+    std::vector<icrar::MVuvw> ToUVWVector(const Eigen::MatrixXd& value)
+    {
+        auto res = std::vector<icrar::MVuvw>();
+        res.reserve(value.rows());
+
+        for(int row = 0; row < value.rows(); ++row)
+        {
+            auto vector = value(row, Eigen::all);
+            res.push_back(icrar::MVuvw(vector(0), vector(1), vector(2)));
```

Comment:
Should be possible to do `emplace_back(vector(0), vector(1), vector(2))` to avoid extra object constructions